cmd/swarmctl/service/update.go

```diff
@@ -36,6 +36,16 @@ var (
 				return err
 			}
 
+			// change the image if that positional arg is present
+			if len(args) > 1 {
+				spec.Task.GetContainer().Image = args[1]
+			}
+
+			// change the args too, if those positional args are present
```

Comment:
BTW, this is the approach that engine is going with: https://github.com/docker/docker/pull/23946

I guess we should be consistent with that (single string argument that we parse with shlex). But let's wait for that PR to go through the review process before doing anything.